src/styles/components/_input-text.scss

```diff
@@ -5,7 +5,7 @@
 .bsds-input-text-input {
   width: 100%;
   margin-top: space(md);
-  border: space(xxs) solid $neutral-40;
+  border: $border-width-xxs solid $neutral-40;
```

Comment:
we could ask catie if she wants to swap this to $border-color-assertive. text and select have $neutral-40, but checkbox and radio are $neutral-00